src/components/AddBookForm.test.js

```diff
@@ -0,0 +1,30 @@
+import AddBookForm from './AddBookForm'
+import user from '@testing-library/user-event'
+import { getByText, render } from '@testing-library/react'
+
+describe('AddBookForm', () => {
+  const onSubmitMock = jest.fn()
+
+  it('calls onSubmit with correct data and resets form', () => {
+    const { getByLabelText, getByText } = render(
+      <AddBookForm onCreateBook={onSubmitMock} />
```

Comment:
If you changed your addBook function as suggested avove, you'd also have to change it here:

      `<AddBookForm addBook={addBookMock} />`

and then of course rename the mock-function as well. (`addBookMock = jest.fn()`)